src/types.js

```diff
@@ -26,5 +26,25 @@ export const TYPES = {
   reviewGuidelines: {
     name: 'Review Guidelines',
     fileName: 'review_guidelines'
+  },
+  cookiesPolicy: {
```

Comment:
> This type of document applies specifically to cookies on Facebook products.
>
> Based on our hypothetical characterization of contractual document types as _provider_ / _user_ / _object_, it makes sense to create a new document type, because the _object_ changes.